src/download-package.js

```diff
@@ -26,12 +34,17 @@ async function downloadPackage(name, url, range) {
   name = parsed.name;
   let version = parsed.rawSpec;
   let _path = path.join(newTmpDir, 'node_modules', name);
-  // let version = require(path.join(_path, 'package')).version;
+  let defaultBlueprintOverride;
+
+  if (parsed['ember-addon'] && parsed['ember-addon']['defaultBlueprint']) {
+    defaultBlueprintOverride = parsed['ember-addon']['defaultBlueprint'];
```

Comment:
I don't think I see a test exercising this new way to override.

Reply:
Ahh good catch, this isn't the right place to put this

Reply:
I moved it into another module and added unit test for it